### ISSUE TITLE
Image field also creates a text field

### ISSUE THREAD

**Issue by Cannonb4ll** [icebob/vue-form-generator#77]
![Image](http://screenshot.serverbuilds.nl/3B2Y47070C42/Image%202016-09-25%20at%201.57.38%20PM.png)

I am not sure why this is happening, but my code is fairly simple:

```                
schema: {
                    fields: [{
                        type: "image",
                        label: "Avatar",
                        model: "avatar",
                        required: true
                    }]
                },
```

**Comment by icebob** [comment]
It's a feature, because you can set image as a HTTP URL, or browse from computer and the component convert it to data URL.

**Comment by Cannonb4ll** [comment]
There is no feature to make this configurable?

**Comment by lionel-bijaoui** [comment]
Not for now, but we could add it for future version.
In the `input` field (that will be available in v0.5.0), "color" and "range" `inputType` also show the color value if set.
We need to add an option to show or hide it.
In the meantime, you can use CSS to hide that element visually:
```css
input.form-control.link {
    display: none;
}
```

**Comment by icebob** [comment]
[Done](https://github.com/icebob/vue-form-generator/blob/master/dev/schema.js#L227).
Will be available in v0.5.0
